web/src/components/dashboard/SidebarCardFundingNudge.test.tsx

```diff
@@ -46,8 +78,76 @@ describe("<SidebarCardFundingNudge />", () => {
     card = generateSidebarCardContent({ id: fundingNudge });
   });
 
+  const generateTaxFilingUserData = (params: {
```

Comment:
[sand] It looks like this is only being used in one place. If that's the case I think there's some logic in here that isn't necessary and may over-complicate the function.

Reply:
This is still true. I think we can simplify this function.

I'm assuming the intention was to write something in a way that it can be reused if needed. Given this is only being used in one place, I'd opt for making this as simple as possible.

Reply:
[boulder] OK, after pulling up the code locally to look into this further, I'm realizing that a lot of this setup is being copied directly from `TaxAccessStepTwo.test.tsx`. Additionally, there's a lot of logic we're pulling from `TaxAccessStepTwo` into `SidebarCardFundingNudge` without actually writing any tests around it.

I think we should talk about the approach here. We can write this code in a way that's DRY and consolidate our test logic here.